supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -247,19 +248,28 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
 
   private def modifyCommandForDocker(jobCmd: Try[String], localizedInputs: CallInputs): Try[String] = {
     Try {
-      val inputFiles = localizedInputs.filter { case (k,v) => v.wdlType.equals(WdlFileType) }
-      val dockerInputDataVol: Seq[String] = inputFiles.values.map { value =>
-        val limit = value.valueString.lastIndexOf("/")
-        value.valueString.substring(0, limit)
+      val inputFiles = localizedInputs.filter {
+        case (k, v) => v.wdlType == WdlFileType || v.wdlType == WdlArrayType(WdlFileType)
+      }
+      val dockerInputDataVol: Seq[String] = inputFiles.values.flatMap {
+        case file if file.wdlType == WdlFileType =>
```

Comment:
You might be able to use `case WdlFile(filename) =>` and `case WdlArray(WdlArrayType(WdlFileType), fileArray) =>` for better case matching?